### PR TITLE
(PC-7478) Display withdraw conditions

### PR DESCRIPTION
**Pull request by blixit**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-7478

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [x] Made sure my feature is working on the relevant real / virtual devices.
- [x] Written **unit tests** for my feature.
- [x] Added new reusable components to the AppComponents page and communicate to the team on slack
- [x] Added a **screenshot** for UI tickets.
- [x] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
| image |                          |                       |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

### CODE REVIEW

File: src/features/bookings/pages/BookingDetails.tsx [hunk 1]
```diff
@@ -108,15 +108,23 @@ export function BookingDetails() {
...
-        <BookingProperties booking={booking} />
-        {canOpenItinerary && (
-          <OpenItineraryContainer>
-            <Spacer.Column numberOfSpaces={4} />
-            <Separator />
-            <Spacer.Column numberOfSpaces={4} />
-            <SeeItineraryButton openItinerary={openItinerary} />
-          </OpenItineraryContainer>
-        )}
+        <ViewWithPadding>
```
Comment (AnoukHello) [inline src/features/bookings/pages/BookingDetails.tsx:111]:
```suggestion
        <Container>
```

File: src/features/bookings/pages/BookingDetails.tsx [hunk 2]
```diff
@@ -108,15 +108,23 @@ export function BookingDetails() {
...
+              <Separator />
+              <Spacer.Column numberOfSpaces={4} />
+              <SeeItineraryButton openItinerary={openItinerary} />
+            </React.Fragment>
+          )}
+          <Spacer.Column numberOfSpaces={8} />
+          <Typo.Title4>{t`Modalités de retrait`}</Typo.Title4>
+          <Spacer.Column numberOfSpaces={4} />
+          <Typo.Body>
+            {t`Rend toi à l’événement le jour J et présente ton code ! Si tu as besoin de plus d’informations n’hésite pas à aller :`}
```
Comment (AnoukHello) [inline src/features/bookings/pages/BookingDetails.tsx:125]: Je crois que sur la maquette c'est seulement un exemple, ici on voudrait plutôt afficher ce qui se trouver dans le champ `booking.stock.offer.withdrawalDetails`

File: src/features/bookings/pages/BookingDetails.tsx (hunk 1 above)
Comment (blixit) [reply src/features/bookings/pages/BookingDetails.tsx:111]: C'est volontairement que je l'ai appelé comme ça. Tant que je n'ai pas appliqué toute la marge à la page.

File: src/locales/fr/messages.po [hunk 3]
```diff
@@ -964,6 +965,10 @@ msgstr "Rechercher"
 msgid "Redirection en cours"
 msgstr "Redirection en cours"
 
+#: src/features/bookings/pages/BookingDetails.tsx:97
+msgid "Rend toi à l’événement le jour J et présente ton code ! Si tu as besoin de plus d’informations n’hésite pas à aller :"
+msgstr "Rend toi à l’événement le jour J et présente ton code ! Si tu as besoin de plus d’informations n’hésite pas à aller :"
```
Comment (AnoukHello) [inline src/locales/fr/messages.po:970]: Elle n'a plus lieu d'être cette traduction si ?

**Review by AnoukHello — APPROVED** [review]

**Pull request merged**